Meta/CMake/utils.cmake

```diff
@@ -221,3 +221,18 @@ function(download_file url path)
         endif()
     endif()
 endfunction()
+
+function(add_jakt_executable exe_name source_file)
+    set(source ${CMAKE_CURRENT_SOURCE_DIR}/${source_file})
+    get_filename_component(CMD_NAME_JAKT ${source} NAME_WE)
+    add_executable(${exe_name} "${CMD_NAME_JAKT}.cpp")
+    compile_jakt(${source})
+    target_link_libraries(${exe_name} LibC)
+    set_target_properties(${exe_name} PROPERTIES EXCLUDE_FROM_ALL TRUE)
+    install(TARGETS ${exe_name} RUNTIME DESTINATION bin OPTIONAL)
+    serenity_component(
+        ${exe_name}
+        RECOMMENDED
+        TARGETS ${exe_name}
+    )
```

Comment:
I don't think this should be part of this function, not all will be `RECOMMENDED`

Reply:
I have updated `add_jakt_executable` to have an optional 3rd argument to specify `RECOMMENDED`/ `REQUIRED`

I think the CI failure on this push was an intermittent issue with package downloads, not sure how I would rerun?